core/modules/referenceletters/pdf/pdf_rfltr_shipping.modules.php

```diff
@@ -151,15 +151,15 @@ function __construct($db=0)
 		$this->posxqtytoship=$this->page_largeur - $this->marge_droite - 28;
 		$this->posxpuht=$this->page_largeur - $this->marge_droite;
 
-		if (!empty($conf->global->SHIPPING_PDF_DISPLAY_AMOUNT_HT)) {	// Show also the prices
+		if (getDolGlobalString('SHIPPING_PDF_DISPLAY_AMOUNT_HT')) {	// Show also the prices
 			$this->posxweightvol=$this->page_largeur - $this->marge_droite - 118;
 			$this->posxqtyordered=$this->page_largeur - $this->marge_droite - 96;
 			$this->posxqtytoship=$this->page_largeur - $this->marge_droite - 68;
 			$this->posxpuht=$this->page_largeur - $this->marge_droite - 40;
 			$this->posxtotalht=$this->page_largeur - $this->marge_droite - 20;
 		}
 
-		$this->posxpicture=$this->posxweightvol - (empty($conf->global->MAIN_DOCUMENTS_WITH_PICTURE_WIDTH)?20:$conf->global->MAIN_DOCUMENTS_WITH_PICTURE_WIDTH);	// width of images
+		$this->posxpicture=$this->posxweightvol - (!getDolGlobalString('MAIN_DOCUMENTS_WITH_PICTURE_WIDTH')?20: getDolGlobalInt('MAIN_DOCUMENTS_WITH_PICTURE_WIDTH'));	// width of images
```

Comment:
```suggestion
		$this->posxpicture=$this->posxweightvol - (!getDolGlobalString('MAIN_DOCUMENTS_WITH_PICTURE_WIDTH')?20: floatval(getDolGlobalString('MAIN_DOCUMENTS_WITH_PICTURE_WIDTH')));	// width of images
```

ne pas commit a faire dans l'ide car par sur de la saisie

Reply:
ok